.vscode/settings.json

```diff
@@ -3,10 +3,11 @@
     "./langchain",
```

Comment:
I actually don't think this should be in the codebase at all?

Reply:
[This stack overflow answer](https://stackoverflow.com/questions/32964920/should-i-commit-the-vscode-folder-to-source-control#:~:text=Recommendation%20is%20to%20generally%20exclude,a%20commit%20is%20checked%20out) is saying it's good practice to include a couple, for sharing certain settings across devs.